src/Microsoft.Extensions.FileProviders.Physical/PhysicalFilesWatcher.cs

```diff
@@ -48,6 +56,7 @@ public class PhysicalFilesWatcher : IDisposable
             FileSystemWatcher fileSystemWatcher,
             bool pollForChanges)
         {
+            AppContext.TryGetSwitch("Switch.Microsoft.AspNetCore.FileProviders.Issue2808", out _issue2808QuirksMode);
```

Comment:
Please use a meaningful name for this, not a bug number. See https://github.com/aspnet/Home/wiki/Quirks-mode-switches-in-patches

Reply:
🤔 https://github.com/aspnet/Microsoft.Data.Sqlite/pull/477/files

This one is hard to name. Any suggestions? Switch.Microsoft.AspNetCore.FileProviders.DoNotThrowArgumentExceptionForFlakyEvents is the most accurate name I can think of...though it's certainly not concise.

Reply:
`IgnoreEmptyRenameEvents`? The name doesn't have to be pretty, I just don't want them to look like some kind of hacker code... Imagine you're working on an app and see that switch name - what would you even think of it? What is `Issue2808`?

Reply:
Let's got with IgnoreEmptyRenameEvents. Issue2808 is aspnet/Home#2808, but there is doubt this would be helpful to users trying to reason about the switch name